Index: lobby-0.1/src/bin/lobby.sh
===================================================================
--- /dev/null
+++ lobby-0.1/src/bin/lobby.sh
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Copyleft Subin Siby - http://subinsb.com
+
+# /usr/share/lobby
+LOBBY_DIR="/usr/share/lobby"
+
+# Check if Lobby installed
+if [ -f $LOBBY_DIR/installed.conf ]; then
+  URL=$(<$LOBBY_DIR/installed.conf)
+  [[ -x $BROWSER ]] && exec "$BROWSER" "$URL"
+  path=$(which xdg-open || which gnome-open) && exec "$path" "$URL"
+  echo "Can't find browser"
+else
+  # Run Setup
+  x-terminal-emulator -e "$LOBBY_DIR/lobby-install-pre.sh"
+fi
Index: lobby-0.1/src/lobby.desktop
===================================================================
--- /dev/null
+++ lobby-0.1/src/lobby.desktop
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=Lobby
+Comment=Open Lobby In Browser
+Icon=lobby
+Exec=lobby
+Terminal=false
+Type=Application
+Categories=GTK;GNOME;
+StartupNotify=true
Index: lobby-0.1/src/lobby.svg
===================================================================
--- /dev/null
+++ lobby-0.1/src/lobby.svg
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
+
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   width="200"
+   height="200"
+   viewBox="0 0 199.99999 200"
+   id="svg2"
+   version="1.1"
+   inkscape:version="0.91 r13725"
+   sodipodi:docname="favicon.svg">
+  <defs
+     id="defs4" />
+  <sodipodi:namedview
+     id="base"
+     pagecolor="#ffffff"
+     bordercolor="#666666"
+     borderopacity="1.0"
+     inkscape:pageopacity="0.0"
+     inkscape:pageshadow="2"
+     inkscape:zoom="1.979899"
+     inkscape:cx="109.15879"
+     inkscape:cy="93.02162"
+     inkscape:document-units="px"
+     inkscape:current-layer="layer1"
+     showgrid="false"
+     units="px"
+     inkscape:window-width="1327"
+     inkscape:window-height="715"
+     inkscape:window-x="41"
+     inkscape:window-y="27"
+     inkscape:window-maximized="1" />
+  <metadata
+     id="metadata7">
+    <rdf:RDF>
+      <cc:Work
+         rdf:about="">
+        <dc:format>image/svg+xml</dc:format>
+        <dc:type
+           rdf:resource="http://purl.org/dc/dcmitype/StillImage" />
+        <dc:title></dc:title>
+      </cc:Work>
+    </rdf:RDF>
+  </metadata>
+  <g
+     inkscape:label="Layer 1"
+     inkscape:groupmode="layer"
+     id="layer1"
+     transform="translate(0,-852.36216)">
+    <text
+       xml:space="preserve"
+       style="font-style:normal;font-weight:normal;font-size:38.29369736px;line-height:125%;font-family:sans-serif;letter-spacing:0px;word-spacing:0px;fill:#1088d5;fill-opacity:1;stroke:none;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1"
+       x="29.515654"
+       y="1035.292"
+       id="text4136"
+       sodipodi:linespacing="125%"><tspan
+         sodipodi:role="line"
+         id="tspan4138"
+         x="29.515654"
+         y="1035.292"
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:239.33561707px;font-family:Ubuntu;-inkscape-font-specification:Ubuntu;fill:#1088d5;fill-opacity:1">L</tspan></text>
+  </g>
+</svg>
Index: lobby-0.1/src/lobby/lobby-install-pre.sh
===================================================================
--- /dev/null
+++ lobby-0.1/src/lobby/lobby-install-pre.sh
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# /usr/share/lobby
+LOBBY_DIR=$(dirname "$(readlink -f "$0")")
+
+echo 'Welcome to Lobby Setup. Installation requires root access. Please allow it :'
+sudo bash $LOBBY_DIR/lobby-install.sh
Index: lobby-0.1/src/lobby/lobby-install.sh
===================================================================
--- /dev/null
+++ lobby-0.1/src/lobby/lobby-install.sh
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+# Copyleft Subin Siby - http://subinsb.com
+
+# http://lobby.subinsb.com/api/lobby/download/latest
+download_url="http://lobby.subinsb.com/api/lobby/download/latest"
+download_file="/tmp/lobby.zip"
+
+workingDir="$(dirname "$(readlink -f "$0")")/lobby"
+
+# COLORS
+red=`tput setaf 1`
+green=`tput setaf 2`
+NC=`tput sgr0` # No Color
+
+if [ "$EUID" -ne 0 ]
+	then echo "Please run as root"
+	exit
+fi
+
+# Update sources
+echo "'apt-get update' should be ran to ensure availability of dependencies."
+echo "Make sure you have the 'main' and 'universe' repositories enabled."
+echo "Shall I run 'apt-get update' for you (y/n) ?"
+read update
+if [ $update = "y" ]; then
+  apt-get update
+fi
+
+# Check if PHP 7 available
+phpseven=$(dpkg -s php7.0 2>&1 | grep 'no information')
+if [ "$phpseven" == "" ]; then
+  echo "Using PHP 7"
+  depends=("apache2" "mysql-server" "php7.0-mysql" "php7.0-curl" "php7.0-json" "libapache2-mod-php7.0" "wget" "unzip")
+else
+  echo "Using PHP 5"
+  depends=("apache2" "mysql-server" "php5-mysql" "php5-curl" "php5-json" "libapache2-mod-php5" "wget" "unzip")
+fi
+
+for i in "${depends[@]}"
+do
+  PKG_OK=$(dpkg-query -W --showformat='${Status}\n' $i|grep "install ok installed")
+	if [ "" == "$PKG_OK" ]; then
+		echo "The installer needs $i to proceed. Install $i (y/n) ?"
+		read install
+		if [ $install = "y" ]
+			then
+			apt-get -f install $i
+			apt-get install $i
+		else
+      echo "${red}Without $i, Lobby can't run. Exiting Installer${NC}"
+			exit
+		fi
+  else
+    echo -e "${green}$i is installed${NC}"
+	fi
+done
+
+echo "All dependencies of Lobby is met. Shall I continue the installation (y/n) ?"
+read proceed
+if [ $proceed = "n" ]; then
+	exit
+fi
+
+rm "$workingDir/config.php"
+echo "Downloading the latest Lobby version"
+
+wget -d -c -O "$download_file" "$download_url"
+
+echo "Extracting into $workingDir"
+unzip -o /tmp/lobby.zip -d "$workingDir"
+
+echo ""
+echo "Changing Permissions of Directory"
+chown -R root:www-data "$workingDir"
+chmod -R ug+rw "$workingDir"
+chmod -R o+r "$workingDir"
+
+echo ""
+echo "Lobby will be accessible from 'http://lobby.dev'. Do you wish to change the domain ? (y/n)"
+read domainPrompt
+
+if [ $domainPrompt = "n" ]; then
+  domain="http://lobby.dev"
+else
+  echo "Type in the domain. I Recommend 'lobby.dev'"
+  read domain
+	echo "Configuring Lobby to run on domain $domain"
+	cat > /etc/apache2/sites-available/lobby-$domain.conf << EOF
+<VirtualHost *:80>
+	ServerAdmin webmaster@localhost
+	ServerName $domain
+	DocumentRoot $workingDir
+	<Directory $workingDir/>
+		Options Indexes FollowSymLinks
+		AllowOverride All
+		Order allow,deny
+		allow from all
+		Require all granted
+	</Directory>
+	ErrorLog /var/log/apache2/error.log
+	LogLevel warn
+	CustomLog /var/log/apache2/access.log combined
+</VirtualHost>
+EOF
+	echo "127.0.0.1 $domain" >> "/etc/hosts"
+	a2ensite "lobby-$domain"
+	service apache2 restart
+	echo -e "${green}Lobby will be available on http://$domain ${NC}"
+fi
+
+echo ""
+
+echo "Enabling rewrite module"
+a2enmod rewrite
+service apache2 restart
+
+touch /usr/share/lobby/installed.conf
+cat > /usr/share/lobby/installed.conf << EOF
+http://$domain
+EOF
+
+echo -e "${green}Lobby was successfully installed in $workingDir${NC}"
